bases/br_inep_censo_escolar/turma/table_config.yaml

```diff
@@ -0,0 +1,562 @@
+        is_in_staging: True # Bool [True, False], whether the column is in the staging table
+        is_partition: False # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: id_curso_educ_profissional
+        description: Código do curso da educação profissional técnica
+        is_in_staging: True # Bool [True, False], whether the column is in the staging table
+        is_partition: False # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: tipo_turma
+        description: Tipo de atendimento da turma (e.g. Unidade prisional, Classe hospitalar)
+        is_in_staging: True # Bool [True, False], whether the column is in the staging table
+        is_partition: False # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: id_tipo_atividade_1
+        description: Código do tipo de atividade complementar 1
+        is_in_staging: True # Bool [True, False], whether the column is in the staging table
+        is_partition: False # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: id_tipo_atividade_2
+        description: Código do tipo de atividade complementar 2
+        is_in_staging: True # Bool [True, False], whether the column is in the staging table
+        is_partition: False # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: id_tipo_atividade_3
+        description: Código do tipo de atividade complementar 3
+        is_in_staging: True # Bool [True, False], whether the column is in the staging table
+        is_partition: False # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: id_tipo_atividade_4
+        description: Código do tipo de atividade complementar 4
+        is_in_staging: True # Bool [True, False], whether the column is in the staging table
+        is_partition: False # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: id_tipo_atividade_5
+        description: Código do tipo de atividade complementar 5
+        is_in_staging: True # Bool [True, False], whether the column is in the staging table
+        is_partition: False # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: id_tipo_atividade_6
+        description: Código do tipo de atividade complementar 6
+        is_in_staging: True # Bool [True, False], whether the column is in the staging table
+        is_partition: False # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: numero_dias_atividade
+        description: Número de dias da semana onde são realizadas atividades da turma
+        is_in_staging: True # Bool [True, False], whether the column is in the staging table
+        is_partition: False # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: braille
```

Comment:
Por que certas variáveis de tipo tem `tipo` no nome e outras não?

Reply:
As que tem tipo, no geral, qualificam alguma característica ao invés de informar a existência dela. Nesse caso eu vi sentido deixar, por justamente informar um tipo. O detalhe é ter que explicar ao que se refere cada valor na descrição da variável, que é o modelo que eu to adotando.

Reply:
Tá beleza. Vamos tocando assim até implementarmos a arquitetura de tabelas-dicionário.